irc/handlers.go

```diff
@@ -799,6 +799,7 @@ func formatBanForListing(client *Client, key string, info IPBanInfo) string {
 	if info.Duration != 0 {
 		desc = fmt.Sprintf("%s [%s]", desc, info.TimeLeft())
 	}
+	desc = fmt.Sprintf("%s added on [%s]", desc, info.TimeCreated.UTC().Format(time.UnixDate))
```

Comment:
Nice, thanks.

Since consistency is a priority here, this should use `time.RFC1123` instead of `time.UnixDate` :-)